lambda/variables.tf

```diff
@@ -214,3 +214,8 @@ variable "database_name" {
   description = "The name to pass to the create db users lambda. Values are either consignmentapi to create the API users or bastion to create the bastion user"
   default     = ""
 }
+
+variable "batch_size" {
+  description = "The maximum number of messages sent to a lambda from an SQS queue."
+  default     = 1
```

Comment:
I'm wondering if it's a bit confusing to have a variable that's only used by some of the lambdas, but do we want this to be the default?

Actually, does this need to be a parameter at all? We made the timeout a parameter, but that was because it needed to be passed to other modules.

Reply:
I did wonder if the variable was worth it. I did it more out of habit than anything. I've removed it and hard coded it into the lambda files. If it's a massive pain in the future we can change it.